openshift-install-powervs

```diff
@@ -1427,6 +1455,12 @@ function main {
       warn "Enabling forceful destruction option for terraform destroy command"
       FORCE_DESTROY=1
       ;;
+    "-flavor")
+      shift
+      flavortype="$1"
+      process_flavor "$flavortype"
+      vars+=" -var-file ../$varfile -var-file ../automation/compute-vars/$flavortype.tfvars"
```

Comment:
From my last comment in #128
```
What if I give -flavor option before -varfile ? I feel varfile will be "".
```
Not sure why you have closed it.

Reply:
Modified the file

I changed the branch so closed the PR.

Reply:
I am still concerned with add -var-file here. I know this should take precedence always.

Reply:
Modified the file.